src/ArrayBufferFileReader.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @flow
+ */
+'use strict';
+
+const ChunkedFileData = require('./ChunkedFileData');
+const MediaFileReader = require('./MediaFileReader');
+
+import type {
+    LoadCallbackType
+} from './FlowTypes';
+
+class ArrayBufferFileReader extends MediaFileReader {
+    _buffer: ArrayBuffer;
+    _fileData: ChunkedFileData;
+
+    constructor(buffer: ArrayBuffer) {
+        super();
+        this._buffer = buffer;
+        this._fileData = new ChunkedFileData();
+    }
+
+    static canReadFile(file: any): boolean {
+        return typeof ArrayBuffer === 'function' && file instanceof ArrayBuffer
+    }
+
+    _init(callbacks: LoadCallbackType): void {
+        this._size = this._buffer.byteLength;
+        console.log(this._size);
```

Comment:
don't forget to remote this console.log.

Reply:
Ok